src/subscript/ofmvol2csv/ofmvol2csv.py

```diff
@@ -278,18 +307,25 @@ def process_volstr(volstr):
 
     columnnames = extract_columnnames(filelines)
     if not columnnames:
-        raise ValueError("No columns found, one line must start with *DATE")
+        raise ValueError("No columns found, one line must contain *DATE")
     logger.info("Columns found: %s", str(columnnames))
 
-    wellframes = []
-    for wellchunk in split_list(filelines, find_wellstart_indices(filelines))[1:]:
-        # wellchunk zero does not contain data:            --------->        ^^^^
-        wellframe = parse_well(wellchunk, columnnames)
-        if not wellframe.empty:
-            wellframes.append(wellframe)
-    if wellframes:
-        return pd.concat(wellframes, sort=False).sort_index()
-    return pd.DataFrame()
+    # For the OFM syntax with each well in a separate text block:
+    if "WELL" not in columnnames:
+        wellframes = []
+        for wellchunk in split_list(filelines, find_wellstart_indices(filelines))[1:]:
+            # wellchunk zero does not contain data:            --------->        ^^^^
+            wellframe = parse_well(wellchunk, columnnames)
+            if not wellframe.empty:
+                wellframes.append(wellframe)
+        if wellframes:
+            return pd.concat(wellframes, sort=False).sort_index()
+        return pd.DataFrame()
+
+    # For the OFM syntax with WELL as a table attribute:
+    data_start_row = [idx for idx, line in enumerate(filelines) if "WELL" in line][0]
+    data = parse_ofmtable(filelines[data_start_row:], columnnames)
+    return data
```

Comment:
Maybe change to only 1 return statement for readability?

Reply:
Done.